client/src/components/uploads/uploadExtension/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
```

Comment:
Same, maybe we can have a generic upload component but definitely not for this PR

Reply:
agreed!